### PR TITLE
Exhibitors and partners API

### PR DESCRIPTION
**Pull request by jakobivarsson**
- Partner and Tag model
- registration_required field in Event model
- exhibitors and partners serializers and views.

### CODE REVIEW

**Comment by jakobivarsson** [comment]
I will take a look at the tests tomorrow.

**Comment by purmonen** [comment]
The reason why the tests fail is because they are outdated. The exhibitor test case tests serializing a company not an exhibitor. The event_serializer and company_serializer have been removed. The serialisers also take a request as the first argument, however the fake request self.factory.get('/api/events') does not set the HTTP_HOST field so when the image tries to use it causes a dictionary error.

**Comment by purmonen** [comment]
Also, starred expressions like *tags_mappings(exhibitor.tags.all()) is not allowed in Python < 3.5.

**Comment by jakobivarsson** [comment]
hm, are those two empty migrations needed?

**Comment by purmonen** [comment]
Yes, otherwise you have multiple leaf nodes. I guess you ran python3 manage.py --makemigrations merge? That's the command that generated them.

File: fair/models.py [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
+            default="Armada %d" % (date.today().year)
+            )
+    year = models.IntegerField(default=date.today().year)
+
+    def __str__(self):
+        return self.name
+
+
+class Partner(models.Model):
+    name = models.CharField(max_length=50)
```
Comment (jakobivarsson) [inline fair/models.py]: Hm I guess we could do that, but I though most of the partners are not actually going to the fair (e.g. Asana, Zoegas) so having them as a company would not make any sense?

**Comment by jakobivarsson** [comment]
Can you review the changes @purmonen ?

**Pull request merged**